pkg/apb/types.go

```diff
@@ -380,17 +389,17 @@ type ExecutionContext struct {
 	ProxyConfig    *ProxyConfig
 }
 
-// Provisioner defines a function that knows how to provision an apb
-type Provisioner func(si *ServiceInstance, statusUpdate chan<- JobState) (string, *ExtractedCredentials, error)
+// Provisioner - a function that knows how to provision an apb
+type Provisioner func(si *ServiceInstance) (<-chan StatusMessage, error)
```

Comment:
I think we can delete these interfaces with this PR?

Reply:
I've actually been going back and forth on this. I deleted them and then force pushed them back in. I'm trying to test asb_jobs, and the `runFn` should render these moot, so I'll probably pull them again.

Reply:
I don't see Provisioner used anywhere.